packaging/deb/postinst

```diff
@@ -1,3 +1,13 @@
 #!/usr/bin/env sh
 echo "This software may collect information about you and your use of the software, and send that to Microsoft."
 echo "Please visit http://aka.ms/dotnet-cli-eula for more information."
+
+# Run 'dotnet new' to trigger the first time experience to initialize the cache
+echo ."NET Core Tools Telemetry
+
+The .NET Core Tools include a telemetry feature (https://github.com/dotnet/cli/pull/2145) that collects usage information. It’s important that the .NET Team understands how the tools are being used so that we can improve them.
```

Comment:
This is suggested by @bleroy, the link to the PR is on our docs already
https://docs.microsoft.com/en-us/dotnet/articles/core/tools/telemetry

The MD file is there
https://github.com/dotnet/docs/edit/master/docs/core/tools/telemetry.md